File: 217-contains-duplicate/solution2.cpp
// Runtime: 90 ms, faster than 80% of C++ online submissions for Contains Duplicate.
// Memory: 60.82 MB, Beats 73.61% of users with C++

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int n=nums.size();
        for(int i=1;i<n;i++)
        {
            if(nums[i]==nums[i-1]) return true;
        }
        return false;
    }
};

File: 58-length-of-last-word/solution2.py
# Runtime: 38ms, Beats 42.53% of users with Python3
# Memory: 16.60MB, Beats 86.16% of users with Python3

class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        r=len(s)-1
        while(r>=0 and s[r]==" "):
            r-=1
        l=r-1
        while(l>=0 and s[l]!=" "):
            l-=1
        return r-l